mm2src/mm2_main/tests/docker_tests/solana_tests.rs

```diff
@@ -1,10 +1,15 @@
-use crate::docker_tests::docker_tests_common::*;
+use serde_json as json;
+
 use mm2_number::bigdecimal::Zero;
 use mm2_test_helpers::for_tests::{disable_coin, enable_solana_with_tokens, enable_spl, sign_message, verify_message};
 use mm2_test_helpers::structs::{EnableSolanaWithTokensResponse, EnableSplResponse, RpcV2Response, SignatureResponse,
                                 VerificationResponse};
-use serde_json as json;
 
+use crate::docker_tests::docker_tests_common::*;
+
+const SOLANA_CLIENT_URL: &str = "http://localhost:8899";
+
+#[ignore]
```

Comment:
So that ignore was probably some leftover from my previous commits. If I remember correctly that test was failing for some reason and to not block this task I've disabled it for now.

As for the `fill_eth` methods, last time we've discussed to try dump account states and programs into the docker container. This way we wont have to handle separate logic related to the Solana as they work differently and it is not that similar to the typical EVM contracts (It can be found here: .docker/Dockerfile.solana-tests).

Reply:
And unfortunately it's impossible for me right now to determine if my work is working as expected as the tests are not really executing: https://github.com/KomodoPlatform/komodo-defi-framework/actions/runs/10110629981/job/27960913639?pr=2168

```
thread 'main' panicked at 'Reached max attempts for <<{docker:?}>>.', mm2src/mm2_main/tests/docker_tests/docker_tests_common.rs:1155:13
```

The result was the same (failing) with and without #[ignore] that I had on one of my test - its not related to the solana.

Reply:
@shamardy

Reply:
> As for the fill_eth methods, last time we've discussed to try dump account states and programs into the docker container. This way we wont have to handle separate logic related to the Solana as they work differently and it is not that similar to the typical EVM contracts (It can be found here: .docker/Dockerfile.solana-tests). This way we can reduce work needed to handle those tests.
Is it ok for you to do it like that?

It's fine as long as we have accounts/addresses with funds to test with. `solana_coin_send_and_refund_maker_payment` and `solana_coin_send_and_spend_maker_payment` tests among others can't be moved to docker tests now since there are no way to have funds for these tests.

> It can be found here: .docker/Dockerfile.solana-tests

This deploys the swap contract only for now. Sure we can deploy token program the same way, but we also need funds on at least one address to test with, but we will probably need more than one address since tests are run in parallel.

Reply:
> And unfortunately it's impossible for me right now to determine if my work is working as expected as the tests are not really executing: https://github.com/KomodoPlatform/komodo-defi-framework/actions/runs/10110629981/job/27960913639?pr=2168
>
>```
>thread 'main' panicked at 'Reached max attempts for <<{docker:?}>>.', >mm2src/mm2_main/tests/docker_tests/docker_tests_common.rs:1155:13
>```
>
>The result was the same (failing) with and without #[ignore] that I had on one of my test - its not related to the solana.

Ignored the tendermint tests for now in https://github.com/KomodoPlatform/komodo-defi-framework/pull/2168/commits/4277284d9136baf25f154576b13916196a9f47ea, will revert this commit once we resolve this issue from our side.

Reply:
For the record, I am currently attempting to adjust the scripts according to the [suggestions provided](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2168#discussion_r1687887413). However, the most difficult part seems to be even starting the tests as they currently are. I am investigating the issues with the tests. Even though the Geth container started correctly and is accessible on port 7000, I am getting a spam of logs like the following:

```
30 07:04:29, docker_tests_common:167] rpc_clients:854] Transport(JsonRpcError { client_info: "coin: MYCOIN", request: JsonRpcRequest { jsonrpc: "1.0", id: "12", method: "getblockcount", params: [] }, error: Transport("Transport 'http://127.0.0.1:7000/' error: connection closed before message completed") })
```

It's not going too smoothly.